### PR TITLE
added full_location.json

### PR DESCRIPTION
**Pull request by icskzmrinal**
Issue #4 resolved. @thealphadollar please review it

### CODE REVIEW

**Comment by thealphadollar** [comment]
@amrav @icyflame Please look if this is the solution we need.

**Comment by icyflame** [comment]
@thealphadollar @icskzmrinal 

1. So, doing this will trigger the Android Calendar app to show the location instead of just the text? 
2. The file is called `latlong.json` but only has the locations in longer form. So, that's a misnomer which should be fixed.

**Comment by icskzmrinal** [comment]
@icyflame  @thealphadollar 
1) Adding the proper text of location already present on google maps shows the exact location even without adding co-ordinates. 
2) Naming it location.json will work?

**Comment by icyflame** [comment]
@icskzmrinal Great!

You can talk to @thealphadollar for the actual review and merge. He maintains this repository now. 🙂

**Comment by thealphadollar** [comment]
@icskzmrinal Naming it full_location.json would work.

Once you've done that, I'll test it and merge the PR.

**Comment by icskzmrinal** [comment]
@thealphadollar Done

File: add_events.py [hunk 1]
```diff
@@ -83,6 +83,9 @@ def main():
     # Get subjects code and their respective name
     with open('subjects.json') as data_file:    
         subjects = json.load(data_file)
+    # Get latlong of classrooms
+    with open('latlong.json') as data_file:
```
Comment (icyflame) [inline add_events.py]: File name not changed here

Comment (icyflame) [inline add_events.py]: File name not changed here

**Comment by icskzmrinal** [comment]
@thealphadollar done

**Comment by thealphadollar** [comment]
@icyflame Please approve.

**Comment by thealphadollar** [comment]
I've tested this, and it works fine.

**Pull request merged**